### PR TITLE
Update the Algolia library icon 

### PR DESCRIPTION
**Pull request by fletchto99**
The current algolia version is failing to load some resources. This PR updates the algolia client library to fix that issue. It also updates moment to use jsdelivr to be consistent with the other CDN.

Furthermore, the current version of moment has some vulnerabilities.

---
This change is [Reviewable](https://reviewable.io/reviews/louisbarranqueiro/hexo-theme-tranquilpeak/540)

### CODE REVIEW

**Comment by LouisBarranqueiro** [comment]
👋 @fletchto99, sounds good to update these dependencies but please do not include other changes.  Also please respect the PR template, include details about errors, so I can understand and reproduce the problem. I don't want to host the image of Algolia on this theme because it's just another thing to maintain. It's easier to just use the link of their press kit.

**Comment by fletchto99** [comment]
> I don't want to host the image of Algolia on this theme because it's just another thing to maintain.

It looks like their press kit no longer offers a hosted solution but rather provides their images as a `data:image...`

The original PR was actually just meant to fix that broken image (I incorrectly assumed that the image was loaded by the library). I jumped the gun a bit by updating the library versions however I think it might be useful to still update them.

File: layout/_partial/search.ejs [hunk 1]
```diff
@@ -4,7 +4,7 @@
             <span class="close-button"><i class="fa fa-times"></i></span>
             <a href="https://algolia.com" target="_blank" rel="noopener" class="searchby-algolia text-color-light link-unstyled">
                 <span class="searchby-algolia-text text-color-light text-small">by</span>
-                <img class="searchby-algolia-logo" src="https://www.algolia.com/static_assets/images/press/downloads/algolia-light.svg">
+                <img class="searchby-algolia-logo" src="assets/images/logo-algolia-nebula-blue-full.svg">
```
Comment (LouisBarranqueiro) [inline layout/_partial/search.ejs]: I think we are not even required to put their logo now so I will check that out and do the necessary.

Comment (fletchto99) [reply layout/_partial/search.ejs]: Cool. If that's the case then it can probably just be removed and we won't need to worry.

Comment (fletchto99) [reply layout/_partial/search.ejs]: It looks like the logo is still required for OSS: https://www.algolia.com/for-open-source/ (under the FAQ)

**Comment by fletchto99** [comment]
Ah, good catch!

**Comment by itpropro** [comment]
Why don't we directly use the embedded version with
```
data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgNTAgNTAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PGcgZmlsbD0ibm9uZSI+PHBhdGggZD0iTTYuNTAyIDBoMzYuMzc5YzMuNTggMCA2LjUwMiAyLjkyIDYuNTAyIDYuNTI5djM2LjUzYzAgMy41OTYtMi45MDggNi41MjgtNi41MDIgNi41MjhINi41MDJDMi45MjIgNDkuNTg3IDAgNDYuNjY5IDAgNDMuMDZWNi41MTJDMCAyLjkyIDIuOTA1IDAgNi41MDIgMCIgZmlsbD0iIzU0NjhGRiIvPjxwYXRoIGQ9Ik0yOS44MzcgMTEuMDd2LTEuN2MwLTEuMTg5LS45Ni0yLjE1Mi0yLjE0NC0yLjE1MWgtNC45OThhMi4xNDggMi4xNDggMCAwIDAtMi4xNDQgMi4xNXYxLjc0NmMwIC4xOTUuMTguMzMuMzc1LjI4NWExNS41NjQgMTUuNTY0IDAgMCAxIDQuMzUtLjYxNSAxNS44IDE1LjggMCAwIDEgNC4yMDEuNTcuMjkuMjkgMCAwIDAgLjM2LS4yODVNMTYuMTQgMTMuMjk1bC0uODU0LS44NTdhMi4xMzggMi4xMzggMCAwIDAtMy4wMyAwbC0xLjAyMSAxLjAyMmEyLjE0NyAyLjE0NyAwIDAgMCAwIDMuMDRsLjg0Ljg0M2MuMTM1LjEzNC4zMy4xMDMuNDUtLjAzMWExNi40NiAxNi40NiAwIDAgMSAxLjYzNS0xLjkyNiAxNS41MjMgMTUuNTIzIDAgMCAxIDEuOTM1LTEuNjUzYy4xNS0uMDkuMTY0LS4zMDIuMDQ1LS40MzhtOS4xMiA1LjM5OXY3LjM1NWMwIC4yMS4yMjcuMzYyLjQyLjI1Nmw2LjUxMy0zLjM4NGMuMTQ4LS4wNzQuMTkzLS4yNTYuMTItLjQwNWE4LjA5NyA4LjA5NyAwIDAgMC02Ljc1Mi00LjEwN2MtLjE1IDAtLjMuMTItLjMuMjg1bTAgMTcuNzE5Yy01LjQzIDAtOS44NDItNC40MjQtOS44NDItOS44NjhzNC40MTItOS44NjYgOS44NDItOS44NjZjNS40MzIgMCA5Ljg0MiA0LjQyMiA5Ljg0MiA5Ljg2NnMtNC4zOTUgOS44NjgtOS44NDIgOS44NjhtMC0yMy44ODRjLTcuNzEyIDAtMTMuOTY3IDYuMjcyLTEzLjk2NyAxNC4wMTYgMCA3Ljc0NiA2LjI1NSAxNC4wMDQgMTMuOTY3IDE0LjAwNCA3LjcxMiAwIDEzLjk2Ny02LjI3MyAxMy45NjctMTQuMDE4IDAtNy43NDYtNi4yNC0xNC4wMDItMTMuOTY3LTE0LjAwMiIgZmlsbD0iI0ZGRiIvPjwvZz48L3N2Zz4=
```
for faster loading times?

**Pull request merged**